frontend/src/app/middleware/middleware.service.ts

```diff
@@ -48,6 +49,14 @@ export class MiddlewareService {
       .post(`${API_BASE}/claims`, claim, {
         headers: this.headers,
       })
-      .subscribe();
+      .subscribe(
+        response => {},
+        err => {
+          console.log(err);
```

Comment:
Use an error dialog here instead.

https://alyle-ui.firebaseapp.com/components/dialog

Reply:
This can be fixed later. Just keep it in mind. We're good to merge this for now.